README.md

```diff
@@ -2,9 +2,33 @@
 [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/SubstraFoundation/distributed-learning-contributivity/blob/master/run_experiment_on_google_collab.ipynb)
 [![Discuss on Slack](https://img.shields.io/badge/chat-on%20slack-orange)](https://substra.us18.list-manage.com/track/click?e=2effed55c9&id=fa49875322&u=385fa3f9736ea94a1fcca969f)
 
-
 # Exploration of dataset contributivity to a model in collaborative ML projects
 
+- [Exploration of dataset contributivity to a model in collaborative ML projects](#exploration-of-dataset-contributivity-to-a-model-in-collaborative-ml-projects)
```

Comment:
I would remove the first bullet point: `Exploration of dataset contributivity to a model in collaborative ML projects
`

Reply:
Done!